css/default.css

```diff
@@ -92,3 +92,10 @@ td.print-overdue    { font-weight: bold; }
 .table-nonfluid {
    width: auto !important;
 }
+
+/* Markdown */
+.markdown-body blockquote {
```

Comment:
I assumed that we would add the style only to the places that requires text formatting wherein markdown is available as well.